### PR TITLE
Add recording

### LINKED ISSUE #31: Enable recordings
**Is your feature request related to a problem? Please describe.**
no

**Describe the solution you'd like**
Add checkbox to room settings to allow recordings for a meeting. If true enforce agreement on joining of meeting.

**Describe alternatives you've considered**
no

**Additional context**
no

### PR DESCRIPTION
**Pull request by SamuelWei**
# Fixes #31

## Type (Highlight the corresponding type)
- Bugfix
- **Feature**
- Documentation
- Refactoring (e.g. Style updates, Test implementation, etc.)
- Other (please describe):

## Checklist
- [x] Code updated to current develop branch head
- [x] Passes CI checks
- [x] Is a part of an issue
- [x] Tests added for the bugfix or newly implemented feature, describe below why if not
- [x] Changelog is updated
- [x] Documentation of code and features exists

## Changes

- Added recording management

### Screenshots
image

image

image

image

image

### CODE REVIEW

**Comment by SamuelWei** [comment]
Presenatation Player not working, wrong url syntax

File: lang/en/app.php [hunk 1]
```diff
@@ -56,6 +57,7 @@
     'flash' => [
         'client_error' => 'An unknown error occurred in the application!',
         'guests_only' => 'The request can only be made by guests!',
+        'popup_blocked' => 'Your browser has blocked the opening of a new window. Please allow pop-ups to open for this page and try again.',
```
Comment (danielmachill) [inline lang/en/app.php:60]: "Your browser has blocked opening a new window" should be sufficient

File: lang/en/rooms.php [hunk 2]
```diff
@@ -130,6 +130,8 @@
         'file_forbidden' => 'The access to the requested file was forbidden.',
         'file_gone' => 'The file has been deleted in the meantime.',
         'no_new_room' => 'You do not have the necessary permission to create a new room.',
+        'recording_forbidden' => 'The access to the requested recording was forbidden.',
```
Comment (danielmachill) [inline lang/en/rooms.php:133]: "access to the requested recording has been denied"

File: lang/en/rooms.php [hunk 3]
```diff
@@ -254,8 +256,66 @@
     ],
     'only_used_by_authenticated_users' => 'This room can only be used by authenticated users.',
     'placeholder_name' => 'John Doe',
+    'recording_accept' => 'I agree to the recording.',
```
Comment (danielmachill) [inline lang/en/rooms.php:259]: I consent to the recording
I agree with the recording

**Review by danielmachill — COMMENTED** [review]
Not reviewed yet: most parts of frontend, docker files, tests

**Review by danielmachill — APPROVED** [review]

**Pull request merged**